### DISCUSSION TITLE
Node not connecting to the controller directly

### DISCUSSION THREAD

**Discussion by thecobra666** [zwave-js/zwave-js-ui#2252] [Q&A] [answered]
I have a qubino mini dimmer which is 2 meters from the controller.  According to the mesh graph it is not connecting directly to the controller. 

A meter from the mini dimmer there is another powered zwave device which it also doesn't connect to directly. Instead, it's routing via devices on the floor above it.. 

Does anyone know why this is happening, and how would I solve this? (If possible at all)

//edit: Okay found the heal button which seems to do it's magic right. :)

**Comment by robertsLando** [comment, marked as answer]
Yeah healing usually fix such things, it essentially tells controller to update node routing table and other things to improve connectivity :)
